### PR TITLE
Data update and fix common.py

### PR DESCRIPTION
**Pull request by hitoshizuku7**
## Why are these changes needed?

1. Fix the error code bug in common.py
2. Organized all the required answers and removed unnecessary answer files.
3. Update the README with model example answers and pairwise results.

## Related issue number (if applicable)

#33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,25 +130,36 @@ python show_result.py \
...
-## Pairwise win-rate of three Japanese LLMs
-| Model                   | win | loss | tie |  win_rate | loss_rate | win_rate_adjusted |
-|:------------------------|----:|-----:|----:|----------:|----------:|------------------:|
-| rinna-3.6b-ppo          |  87 |   25 |  52 |  0.530488 |  0.152439 |          0.689024 |
-| rinna-3.6b-sft-v2       |  70 |   35 |  58 |  0.429448 |  0.214724 |          0.607362 |
-| japanese-alpaca-lora-7b |  15 |  112 |  38 |  0.090909 |  0.678788 |          0.206061 |
+## Pairwise win-rate compared with GPT-3.5-davinci-003
+| Model                                                    | Win | Loss | Tie | Win Rate | Loss Rate | Win Rate Adjusted |
+|:---------------------------------------------------------|----:|-----:|----:|---------:|----------:|------------------:|
+| llm-jp--llm-jp-13b-instruct-lora-jaster-dolly-oasst-v1.0 |  65 |  146 |  26 | 0.274262 | 0.616034  |          0.329114 |
```
Comment (hkiyomaru) [inline README.md:152]: This line is weird, as we just have 80 questions.

File: configs/cyberagent--calm2-7b-chat.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "model_name_or_path": "cyberagent/calm2-7b-chat",
   "lora_model_name_or_path": null,
   "tokenizer_name_or_path": null,
-  "prompt_template": "USER: {instruction}\nASSISTANT: ",
+  "prompt_template": "USER:{instruction} ASSISTANT:",
```
Comment (hkiyomaru) [inline configs/cyberagent--calm2-7b-chat.json]: Why did you make this change? I followed [the official instructions](https://huggingface.co/cyberagent/calm2-7b-chat#usage).

Comment (hitoshizuku7) [reply configs/cyberagent--calm2-7b-chat.json]: > Why did you make this change? I followed [the official instructions](https://huggingface.co/cyberagent/calm2-7b-chat#usage).

Oh, it is a mistake I made. I think this change is unnecessary.

File: README.md (hunk 1 above)
Comment (hitoshizuku7) [reply README.md:152]: > This line is weird, as we just have 80 questions.

I did the comparison llm-jp--llm-jp-13b-instruct-lora-jaster-dolly-oasst-v1.0 and text-davinci-003 three times. So there are more than three times as many results. I think we should use the average result of 3 times or just select 80 results from a one-time comparison.

File: configs/cyberagent--calm2-7b-chat.json (hunk 2 above)
Comment (hkiyomaru) [reply configs/cyberagent--calm2-7b-chat.json]: Okay, then please revert this commit.

File: README.md (hunk 1 above)
Comment (hkiyomaru) [reply README.md:152]: As with the other models, a single evaluation result is fine, I think.

**Pull request merged**